![image](https://github.com/user-attachments/assets/da7ac4ac-b3b4-4158-b5a9-6844cacd3750)---
layout: page
title:
permalink: /publications
---

<h1 align="center">Publications</h1>

### 2025

T Hui#, J Zhou#, M Yao, Y Xie, **H Zeng**\*, Advances in spatial omics technologies. Small Methods (2025), e2401171.
D Bai#, J Yang#, X Xue#, Y Gao#, Y Wang, M Cui, B He, **H Zeng**, H Xiang, Z Guo, L Zhu, J Gao, C Zhu\*, F Tang\*, C Yi\*, Single-cell 5-hydroxymethylcytosine landscapes of mouse early embryos at single-base resolution. Cell Rep. (2025) 44, 115520.

---

### 2024

Z Tang#, S Luo#, **H Zeng**, J Huang, X Sui, M Wu, X Wang\*, Search and match across spatial omics samples at single-cell resolution. Nat. Methods (2024) 21, 1818–1829.

---

### 2023

**H Zeng#**, J Huang#, J Ren#, C K Wang, Z Tang, H Zhou, Y Zhou, H Shi, A Aditham, X Sui, H Chen, J A Lo, X Wang\*. [Spatially Resolved single-cell translatomics at molecular resolution.](https://www.science.org/doi/10.1126/science.add3067)  **Science** (2023) 380, eadd3067. (Highlighted in the  Nature Reviews Genetics and  Nature Methods ).[get the PDF]({{ site.url }}/assets/science.add3067.pdf)


J Ren#, H Zhou#, **H Zeng#**, C K Wang, J Huang, X Qiu, X Sui, Q Li, X Wu, Z Lin, J A Lo, K Maher, Y He, X Tang, J Lam, H Chen, B Li, D E Fisher, J Liu, X Wang\*. [Spatiotemporally resolved transcriptomics reveals the subcellular RNA kinetic landscape.](https://www.nature.com/articles/s41592-023-01829-8)  **Nat. Methods** (2023) 20, 695–705.
[get the PDF]({{ site.url }}/assets/s41592-023-01829-8.pdf)


**H Zeng#**, J Huang#, H Zhou#, W J. Meilandt, B Dejanovic, Y Zhou, C J. Bohlen, S Lee, J Ren, A Liu, Z Tang, H Sheng, J Liu, M Sheng \*, X Wang\*. [Integrative in situ mapping of single-cell transcriptional states and tissue histopathology in a mouse model of Alzheimer's disease.](https://www.nature.com/articles/s41593-022-01251-x)  **Nat. Neurosci.** (2023) 26, 430–446.[get the PDF]({{ site.url }}/assets/s41593-022-01251-x.pdf)

---

### Before 2023

B He#, C Zhang#, X Zhang#, Y Fan#, **H Zeng**, J Liu, H Meng, D Bai, J Peng, Q Zhang\*, W Tao\*, C Yi\*. [Tissue-specific 5-hydroxymethylcytosine landscape of the human genome.](https://www.nature.com/articles/s41467-021-24425-w)  **Nat. Commun.** (2021) 12:4249.[get the PDF]({{ site.url }}/assets/2021_HeBo--NC.pdf)

M Liu#, J Zhang#, C Zhu#, X Zhang, W Xiao, Y Yan, L Liu, **H Zeng**, YQ Gao\*, C Yi\*. [DNA repair glycosylase hNEIL1 triages damaged bases via competing interaction modes.](https://www.nature.com/articles/s41467-021-24431-y)  **Nat. Commun.** (2021) 12:4108.[get the PDF]({{ site.url }}/assets/2021_LiuMenghao--NC.pdf)

A Aditham#, H Shi#, J Guo, **H Zeng**, Y Zhou, S D Wade, J Huang, J Liu, X Wang\*. [Chemically modified mocRNAs for highly efficient protein expression in mammalian cells.](https://pubs.acs.org/doi/10.1021/acschembio.1c00569)  **ACS Chem. Biol.** (2022) 17, 3352-3366.[get the PDF]({{ site.url }}/assets/2022_Abhishek Aditham--ACS Chemical Biology.pdf)

**H Zeng#**, M Mondal#, R Song#, J Zhang, B Xia, M Liu, C Zhu, B He, YQ Gao\* and C Yi\*. [Unnatural cytosine bases recognized as thymines by DNA polymerases via the formation of the Watson-Crick geometry.](https://onlinelibrary.wiley.com/doi/10.1002/anie.201807845)  **Angew. Chem. Int. Ed.** (2019) 58, 130-133.[get the PDF]({{ site.url }}/assets/2019_ZengHu--AngewChemIntEdit.pdf)

**H Zeng#**, B He#, C Yi\*. [Compilation of modern technologies to map genome-wide cytosine modifications in DNA.](https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.201900035)  **Chembiochem**  (2019) 20, 1898-1905.(Review)[get the PDF]({{ site.url }}/assets/2019_ZengHu_HeBo--ChemBioChem.pdf)

**H Zeng#**, B He#, B Xia, D Bai, X Lu, J Cai, L Chen, A Zhou, C Zhu, H Meng, Y Gao, H Guo, C He\*, Q Dai\*, and C Yi\*. [Bisulfite-Free, Nanoscale Analysis of 5-Hydroxymethylcytosine at Single Base Resolution.](https://pubs.acs.org/doi/10.1021/jacs.8b08297)  **J. Am. Chem. Soc.** (2018) 140, 13190-13194.[get the PDF]({{ site.url }}/assets/2018_ZengHu--JACS.pdf)

**H Zeng#**, B He#, C Yi\*, and J Peng\*. [Liquid biopsies: DNA methylation analyses in circulating cell-free DNA.](https://www.sciencedirect.com/science/article/abs/pii/S1673852718300432?via%3Dihub)  **J. Genet. Genomics** (2018) 45, 185-192. (Review)[get the PDF]({{ site.url }}/assets/2018_ZengHu_HeBo--JGG.pdf)

C Zhu#, Y Gao#, H Guo#, B Xia#, J Song, X Wu, **H Zeng**, K Kee, F Tang\*, and C Yi\*. [Single-Cell 5-Formylcytosine Landscapes of Mammalian Early Embryos and ESCs at Single-Base Resolution.](https://www.cell.com/cell-stem-cell/fulltext/S1934-5909(17)30070-X)  **Cell Stem Cell** (2017) 20, 720-731 e725.[get the PDF]({{ site.url }}/assets/2017_ZhuC_GaoY_GuoH_XiaB--CellStemCell.pdf)

B Xia#, D Han#, X Lu#, Z Sun, A Zhou, Q Yin, **H Zeng**, M Liu, X Jiang, W Xie, C He\*, and C Yi\*. [Bisulfite-free, base-resolution analysis of 5-formylcytosine at the genome scale.](https://www.nature.com/articles/nmeth.3569)  **Nat. Methods** (2015) 12, 1047-1050.[get the PDF]({{ site.url }}/assets/2015_XiaBo_HanD_LuX--NatMethods.pdf)
